src/olympia/reviewers/views.py

```diff
@@ -770,6 +783,8 @@ def review(request, addon, channel=None):
         user_ratings=user_ratings,
         version=version,
         VERSION_ADU_LIMIT=VERSION_ADU_LIMIT,
+        versions_that_would_be_enabled=versions_that_would_be_enabled,
+        VERSIONS_THAT_WOULD_BE_ENABLED_MAX=10,
```

Comment:
Noted in the other comment I'd throw that on a setting not only for the testability but so it is with the other settings.